chapter_unit_test_first_view.asciidoc

```diff
@@ -727,6 +729,8 @@ class HomePageTest(TestCase):
 // reading experience if the test method we've already seen was replaced by a `...`. Though
 // it probably would need an explanation that it's a convention we're using.
 
+// SEBASTIAN: Also, I'd suggest adding an import at the top of the snippet `from django.test import TestCase`
```

Comment:
thanks.  django puts that import into tests.py for you by default, i think that's why i didn't bother including it...